docs/get-started/onnx/index.md

```diff
@@ -0,0 +1,249 @@
+# Using ONNX to deploy a finetuned model
+
+Imagine a scenario where you successfully used Finetuner to finetune a model on your search problem using data from the respective domain. What comes next?
+
+Naturally, you would want to deploy your embedding model in a service and use it to
+encode data as part of a bigger search application. Jina's [core framework](https://docs.jina.ai/)
+provides the infrastructure layer to help you build and deploy neural search components.
+By implementing custom executors or using existing ones from [Jina Hub](https://hub.jina.ai/)
+you can define the building blocks of your search application and glue everything
+together in a fully-fledged search pipeline.
+
+To use your model with Jina, you would have to build a new executor tailored to your model and upload it to the hub. You can then use a Jina Flow to deploy your search
+application locally or in the cloud using kubernetes.
+
+It is difficult to provide a unified executor that can support all models trained with finetuner,
+since finetuner supports different computational frameworks and each model differs in terms
+of pre-processing required, input type etc.
+
+There is an alternative though, that can be useful for standardizing your deployment
+procedure and unifying different models. Finetuner allows the user to use ONNX for converting
+finetuned models to the ONNX format, whether these models have been trained with PyTorch,
+TensorFlow or PaddlePaddle. The ONNX format is an open standard for defining
+neural network architectures. You can find out more in the [ONNX webpage](https://onnx.ai/).
+
+After converting a trained model to the ONNX format, you can use the
+[ONNX encoder](https://hub.jina.ai/executor/2cuinbko) from Jina Hub to deploy your embedding
+model. The encoder simply loads your embedding model in ONNX format and uses the ONNX
+runtime for running inference. The document tensors are fed to the ONNX model as input
+and the output NumPy vectors are assigned to the documents as embeddings.
+
+The ONNX encoder takes care of the embedding part and the only thing left is to add a custom
+executor for pre-processing in case there is the need for one. Additionally, you gain
+a boost in efficiency, since the conversion to ONNX already optimizes the model in various
+ways.
```

Comment:
optimize the model based on the platform/hardware device? can you rephrase this a bit?